src/algebra/group/units.lean

```diff
@@ -128,6 +131,27 @@ section monoid
 
 end monoid
 
+section comm_monoid
+
+variables [comm_monoid α]
+
+theorem divp_eq_divp_of {x y : α} {ux uy : units α} (h : x * uy = y * ux) :
```

Comment:
I think the trailing `_of` is not that common, for example compare `div_eq_div` from `algebra.group.with_one`.

Reply:
Deleted this lemma.